OpenRA.Mods.Cnc/Traits/Minelayer.cs

```diff
@@ -249,7 +252,7 @@ protected override IEnumerable<IRenderable> RenderAboveShroud(WorldRenderer wr,
 
 			protected override string GetCursor(World world, CPos cell, int2 worldPixel, MouseInput mi)
 			{
-				return "ability";
+				return minelayers.First().Trait<Minelayer>().Info.Cursor;
```

Comment:
Please assign a `readonly string cursor` in the constructor like we do for the overlay tiles.

Reply:
https://github.com/OpenRA/OpenRA/pull/17843/ already solved this better, so I would like that get merged first.